Pilot.gyp

```diff
@@ -0,0 +1,199 @@
+{
+  'variables': {
+    'pilot_core_sources': [
```

Comment:
Could we leverage more `*.swift` in all these source lists? I'd prefer to not have to have folks add source specifically here in addition to Xcode targets.

Reply:
Done.